user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -59,6 +59,9 @@ Forge
 Others
 ------
 
+- **numberNative:** Added the ``numberNative`` attribute to the Database setting to keep the variable type obtained after SQL Query consistent with the type set in the DataBase.
```

Comment:
```suggestion
- **Database:** Added the ``numberNative`` attribute to the Database Config for MySQLi to keep the variable type obtained after SQL Query consistent with the type set in the datadase.
```

Reply:
Thank you, I have revised this description.